pkg/controllers/provisioning/scheduling/inflightnode.go

```diff
@@ -35,20 +35,23 @@ type InFlightNode struct {
 	requirements       scheduling.Requirements
 	available          v1.ResourceList
 	startupTolerations []v1.Toleration
-	hostPortUsage      *state.HostPortUsage
+	hostPortUsage      *scheduling.HostPortUsage
```

Comment:
Why did we move these into the top level `scheduling` package? Seems like it may be better encapsulated in this package?

Reply:
They're both used in pkg/controllers/state/ and pkg/controllers/provisioning/scheduling/ so I thought they fit better at the higher level pkg/scheduling/.